tf2_ros/test/test_buffer_client.cpp

```diff
@@ -61,7 +61,13 @@ class MockBufferServer : public rclcpp::Node
       [](const rclcpp_action::GoalUUID &, std::shared_ptr<const LookupTransformAction::Goal>)
         {return rclcpp_action::GoalResponse::ACCEPT_AND_EXECUTE;},
       [](const std::shared_ptr<GoalHandle>){return rclcpp_action::CancelResponse::ACCEPT;},
-      std::bind(&MockBufferServer::acceptedCallback, this, std::placeholders::_1));
+      std::bind(&MockBufferServer::handle_accepted, this, std::placeholders::_1));
+  }
+
+  void handle_accepted(const std::shared_ptr<GoalHandle> goal_handle)
```

Comment:
Style nit: all of the other methods in here are camelCase, I think this should be `handleAccepted`.